react/components/PriceRange/index.js

```diff
@@ -20,13 +23,28 @@ const PriceRange = ({ title, facets, priceRange, priceRangeLayout }) => {
   const intl = useIntl()
   const navigateTimeoutId = useRef()
 
+  const { push } = usePixel()
+  const { searchQuery } = useSearchPage()
+
+  const pushPixelEvent = (left, right) => {
+    push({
+      event: 'filterManipulation',
+      items: {
+        filterProductCategory: getCategoryFromObjs(searchQuery.products),
+        filterName: 'PriceRange',
+        filterValue: `[${left.toString()}-${right.toString()}]`,
+      },
+    })
+  }
+
   const { fuzzy, operator, searchState } = useSearchState()
 
   const handleChange = ([left, right]) => {
     if (navigateTimeoutId.current) {
       clearTimeout(navigateTimeoutId.current)
     }
 
+    pushPixelEvent(left, right)
```

Comment:
Why not use the function you exported? It supports this case and avoids code duplication

Reply:
I agree with you, changed using the pushPixelEvent function